### PR TITLE
Now uses http to load translations and bundles in tests

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: src/__mocks__/react-i18next.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  });
+};
+
+import common from "../../public/locales/en/common.json";
+import clients from "../../public/locales/en/clients.json";
+import realm from "../../public/locales/en/realm.json";
+import groups from "../../public/locales/en/groups.json";
+import help from "../../public/locales/en/help.json";
+
+const translation = { ...common, ...help, ...clients, ...realm, ...groups };
```
Comment (ssilvert) [inline src/__mocks__/react-i18next.js:43]: So if I understand, all components running inside tests will now be able to access `useTranlation` without any additional setup except to add new bundles here?

Comment (edewit) [reply src/__mocks__/react-i18next.js:43]: yes that is right

Comment (ssilvert) [reply src/__mocks__/react-i18next.js]: @edewit Perfect.  

Looks like we still have test failures.  Please fix so we can get this merged.

**Comment by edewit** [comment]
@ssilvert already tried to make the test not fail for the last 4 commits of this PR

**Comment by ssilvert** [comment]
> @ssilvert already tried to make the test not fail for the last 4 commits of this PR

OK.  Do you want to try some more or just close this PR?  We can come back and do this later if you want.

**Pull request closed without merging**